internal/pkg/override/patch.go

```diff
@@ -78,6 +92,7 @@ func (p *Patch) Override(body []byte) ([]byte, error) {
 }
 
 func unmarshalPatches(path string, fs afero.Fs) ([]yamlPatch, error) {
+	path = filepath.Join(path, "cfn.patches.yml")
```

Comment:
nit: can we move this to a const in this file? It seems important enough to highlight it

Reply:
![](https://media.tenor.com/s1w62ls6xlQAAAAC/princess-bride.gif)

Reply:
lmao